SFBaseKitDemo/SFBaseKitDemo/ExampleHomeModule/HomeCoordinator.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  HomeCoordinator.swift
+//  SFBaseKitDemo
+//
+//  Created by Aleksandar Geyman on 21.04.20.
+//  Copyright © 2020 Upnetix. All rights reserved.
+//
+
+import SFBaseKit
+
+protocol HomeSceneDelegate: Coordinator {
+    
```

Comment:
New functionality request - Add logout method in this protocol.

Reply:
Done.